docs/api/create_docker_container.md

```diff
@@ -0,0 +1,78 @@
+# Creating a container
+
+Testcontainers' generic container support offers the most flexibility, and makes it easy to use virtually any container images as temporary test environments. To interact or exchange data with a container, Testcontainers for .NET provides an implementation of the `TestcontainersBuilder<TDockerContainer>` to configure and create the resource.
+
+## Examples
+
+A NGINX container that binds the HTTP port and hosts static content. Hit the static content in your browser at `http://localhost:80` or `http://host-ip:80`.
```

Comment:
I would suggest to not advocate for fixed port usage in the docs (not even document it). Part of Testcontainers is ensuring best-practices and in our messaging, we are generally fairly strong on using dynamic ports.

Reply:
Regarding this and the below example, it is easier for the developer to interact with a fixed port. Otherwise we need to explain how to look up the random port mapping to test it in a browser or via telnet.

Reply:
But that is also how we do it in Java. I agree that it is easier to cater to the existing mindset (devs are used to fixed ports), but we learned when working with the Testcontainers for Java community, that breaking the mentality of fixed ports requires some effort and I would like us to double down on clearly communicating this mindset. Else we will have to investest time to change habits in the future. Indeed in the case of Java, we deprecated our `FixedHostPortContainer` implementation (and plan to remove it eventually). 

This is the corresponding section in Testcontainers that explains how to use dynamic ports, I think it is sufficiently accessible: 
https://www.testcontainers.org/features/networking/

Reply:
I've updated both examples.